dashboard/db/migrate/20250305163452_create_user_level_evaluations.rb

```diff
@@ -0,0 +1,19 @@
+class CreateUserLevelEvaluations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :user_level_evaluations do |t|
+      t.integer :user_id, null: false
+      t.integer :level_id, null: false
+      t.integer :script_id, null: false
+      t.string :school_year, null: false
+      t.text :evaluation_criteria
+      t.text :ai_evaluation
+      t.text :ai_reasoning
+      t.string :ai_model_version
+
+      t.timestamps
+
+      t.index :user_id
```

Comment:
keep an eye out for how you access this data. if you end up using it like we use user_levels, then you may want to augment this index to be more like https://github.com/code-dot-org/code-dot-org/blob/158478996fc1f1981bab749d598af2b691de662e/dashboard/app/models/user_level.rb#L23